style/book.css

```diff
@@ -171,7 +177,7 @@ div.indented {
 .md img {
     margin-top: 1.0em;
     width: 72ex;
-    //image-rendering: crips-edges;
```

Comment:
Should be "crisp-edges".

Also, @armansito — if you look at the RTW Markdeep source, search for the string "class='pixel'". This image (or figure) class ensures that images are rendered in a pixelated fashion without edge smoothing. We generally use this class when showing render results, but _don't_ use it for things like figures and other drawings. You may want to follow the same approach instead of rendering every image in pixelated fashion.